lib/validation-stream.js

```diff
@@ -160,6 +172,15 @@ extend(ValidationStream.prototype, {
   },
   _read: function (size) {
     // DEV: Do not take any action as we only emit one data event, the result
+  },
+  _emitEmptyCssEvents: function () {
+    var data = {
+      validity: true,
+      errors: [],
+      warnings: []
+    };
+    this.emit('data', data);
+    this.emit('end');
```

Comment:
While streams are event emitters, we should stick to their built-in API. We should use `push(result)` and `push(null)`:

https://github.com/twolfson/css-validator/blob/0.7.0/lib/validation-stream.js#L150-L152

https://nodejs.org/api/stream.html#stream_readable_push_chunk_encoding